docker-compose.yml

```diff
@@ -1,19 +1,70 @@
 services:
-  mysql:
+  mysql_auth:
```

Comment:
would be far better to name it ``auth_mysql`` so we can group thigns together easier

Reply:
done